### PR TITLE
Revert Operator CI

### LINKED ISSUE #11206: Operator CI is broken reporting docker login failures
### Describe the bug

The Operator CI workflow is reporting login failures while running `docker login`. 

### Version

17.0.1

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

A few examples: 

- https://github.com/keycloak/keycloak/runs/5890690824?check_suite_focus=true
- https://github.com/keycloak/keycloak/runs/5892142025?check_suite_focus=true
- https://github.com/keycloak/keycloak/runs/5887096987?check_suite_focus=true

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by vmuzikar**
Fixes https://github.com/keycloak/keycloak/issues/11206.

### CODE REVIEW

**Review by abstractj — APPROVED** [review]
@vmuzikar approving, but I will wait for @andreaTP to review.

**Comment by andreaTP** [comment]
Given that the actual fix does require some more changes and workarounds in the CI pipeline I approve this one to unblock the rest of the contributors 👍

**Review by andreaTP — APPROVED** [review]

**Pull request merged**
